scss/components/_feedback.scss

```diff
@@ -0,0 +1,137 @@
+@import "../variables";
+
+.feedback {
+  background: $color-primary;
+  padding-top: 100px;
+  padding-bottom: 40px;
+}
+
+.feedback-headline {
+  font-size: 45px;
+  flex-direction: column;
+  display: flex;
+}
+
+.feedback-underline {
+  margin: 17px auto 0 136px;
+
+  &-headline {
+    font-weight: bold;
+    margin: 45px 0 0 134px;
+  }
+
+  &-text {
+    margin: 20px 0 0 135px;
+    font-size: 18px;
+  }
+}
+
+.connection {
+  padding: 57px 0 0 153px;
+  display: flex;
+  flex-direction: row;
+}
+
+.connection-list {
+  display: flex;
+  width: 25%;
+  flex-direction: column;
+  flex-wrap: wrap;
+}
+
+.typing {
+  width: 61%;
+  margin-left: 60px;
+  line-height: 66px;
+}
+
+.list-item {
+  font-size: 14px;
+  display: flex;
+  padding-bottom: 49px;
+  flex-direction: row;
+
+  &:hover {
+    color: $second-color;
+    cursor: pointer;
+  }
+}
+
+.special-center-text {
+  margin: 14px 0 0 65px;
+}
+
+.email-image {
+  height: 19px;
+  margin: 19px 0 0 12px;
+}
+
+.special-bot-text {
+  padding-top: 15px;
+}
+
+.phone-image {
+  margin: 35px 0 0 11px;
+}
+
+.list-item-location {
+  margin: 21px 0 0 11px;
+  width: 24px;
+  height: 35px;
+}
+
+.list-item-text {
+  margin: 0 0 0 65px;
+  line-height: 25px;
+}
+
+.name {
+  width: $container-width * 0.243;
+  font-style: italic;
```

Comment:
Extract common input's styles to `input-box` class, for example `font-style`, `padding `, `height`.
You should follow the DRY principle.

Reply:
Understood, I will follow that